### ISSUE TITLE
Kitten Themes UI clarity.

### ISSUE THREAD

**Issue by AdamEkelof** [kovidgoyal/kitty#7276]
**Describe the bug**
When changing the theme in kittens themes you are met with a page asking whether you want to modify, place, abort or quit. These 4 alternatives are formated as [M]odify, [P]lace, [A]bort and [Q]uit. But even though they use capital letters, the ui does not accept capital letters. 

**To Reproduce**
1. run kitten themes
2. select a theme
3. press M, P, A or Q (not m, p, a or q).

Basically, add the option to use capital letters, or change the prompt to [m]odify, [p]lace, [a]bort and [q]uit.
